### PR TITLE
Revert "Added geolocation tool - Astrometry"

### PR DESCRIPTION
**Pull request by Invisibleman9**
Reverts lockfale/OSINT-Framework#302

### CODE REVIEW

**Comment by s0lray** [comment]
Why revert this change?

**Comment by Invisibleman9** [comment]
Sorry mistake on me

**Review by Invisibleman9 — COMMENTED** [review]
Didn't mean to revert

**Review by Invisibleman9 — COMMENTED** [review]
I tried to undo the revert

**Comment by Invisibleman9** [comment]
Sorry I’m a newbie I didn’t mean to I appreciate your help thank you

File: public/arf.json [hunk 1]
```diff
@@ -1,4 +1,4 @@
-{
```
Comment (Invisibleman9) [inline public/arf.json:1]: Kaylapadgett4@icloud.com <7068895599>

**Review by Invisibleman9 — COMMENTED** [review]
Cool

**Pull request closed without merging**